.circleci/config.yml

```diff
@@ -14,4 +14,5 @@ jobs:
         command: |
           if [ "${CIRCLE_BRANCH}" == "master" ]; then
             ./architect deploy
+            ./architect publish --stable
```

Comment:
@fgimenez @indyfree is this good enough or is more magic required?

Reply:
We usually have a seperate job for publishing to `stable` defined. And publish to `beta` with the deploy job. In the hinsight of easier release phasing in the future. Like here:
https://github.com/giantswarm/kubernetes-nginx-ingress-controller/blob/ab6f1b9e8fc68af22e97293b6c0141615c64063c/.circleci/config.yml#L66-88

But I guess it's ok for now. We would need to touch here anyway if we introduce something like that.

Reply:
What is the thing with `stable` and `beta` why the difference? Why only these two? I am fine with iterating here and get it working in the first place. I am not a fan of the circle configs anyway.

Reply:
Exactly, the idea behind having separate jobs was making it easier introduce later more testing jobs in between so that we can publish to stable confidently. Given that there are no tests yet, deploying and publishing to stable in the same job should be ok.

Reply:
Currently there is no difference. These two channels point to the same release. We specced it in that way, so we could introduce a release promotion. See here: https://github.com/giantswarm/giantswarm/blob/master/roadmap/MANAGED-SERVICES.md#release-promotion

But there is no implementation in that direction currently.

Reply:
In fact we are not using beta right now AFAIK, we specify a temporary channel explicitly for testing like here https://github.com/giantswarm/kubernetes-nginx-ingress-controller/blob/ab6f1b9e8fc68af22e97293b6c0141615c64063c/.circleci/config.yml#L22. Beta should be used in the tests before pushing to stable according to the spec.